app/controllers/users/sessions_controller.rb

```diff
@@ -91,13 +93,21 @@ def cache_active_profile
     end
 
     def user_signed_in_and_not_locked_out?(user)
-      return false unless current_user.present?
-
+      return false unless current_user
       !user_locked_out?(user)
```

Comment:
based on the name of this method, I'd expect this to read `current_user && !user_locked_out?(user)`